src/main/java/com/cloudogu/scm/review/config/api/PullRequestConfigDto.java

```diff
@@ -41,4 +41,11 @@ public PullRequestConfigDto(Links links) {
   private boolean preventMergeFromAuthor;
   private boolean restrictBranchWriteAccess;
   private List<String> protectedBranchPatterns;
+  private List<ExceptionEntryDto> protectedBranchExceptions;
+
+  @Getter @Setter
+  public static class ExceptionEntryDto {
```

Comment:
I was confused for a second why this is named exception. I can't think of a better name, though.

Reply:
`ProtectionBypass` or simply `bypass` it will be...